### PR TITLE
Experience-8026: Widen grid container

### PR DESCRIPTION
**Pull request by stephenkao**
This changeset updates the follow pages' containers to be widescreen (up to 1400px) since they're the most data-heavy:
- Daily Data
- Submissions
- Message Tracker

The Public Validator page will continue to be on 1024px to align with the rest of the public pages.

Additionally, there's some cleanup regarding containers in the codebase across several different pages, and I've updated `grid-container` modifier classes to use the GridContainer component instead.  There are a lot of `grid-container` modifier classes used around the codebase, and it really shouldn't be used this much; it should be used at the highest ancestor node of common width, and all descendants should naturally expand to their own widths inside of that.  The grid container sets a maximum width, yes, but it also includes padding to account for grid gutters -- if this is used too frequently, there'll be a lot of horizontal jaggedness across the website.

**tl;dr In most cases, a grid container should be used at the top of the page and nowhere else.**  There are obviously use cases for nested grid containers, but they should be rare and should be brought up by design rather than used anywhere we need a maximum width -- especially not in components that should be able to be rendered agnostically to the wrapping context.

Daily Data:
Screenshot 2023-03-13 at 10 43 58
Screenshot 2023-03-13 at 10 44 38

Submissions:
Screenshot 2023-03-13 at 10 44 08
Screenshot 2023-03-13 at 10 44 25

Message Tracker:
Screenshot 2023-03-13 at 10 44 52
Screenshot 2023-03-13 at 10 44 57

### CODE REVIEW

File: frontend-react/src/components/FileHandlers/FileHandler.tsx [hunk 1]
```diff
@@ -260,82 +261,86 @@ function FileHandler() {
     }
 
     return (
-        <div className="grid-container usa-section margin-bottom-10">
```
Comment (stephenkao) [inline frontend-react/src/components/FileHandlers/FileHandler.tsx:263]: No markup changes here until than wrapping in a GridContainer + article.

File: frontend-react/src/components/Admin/AdminReceiverDashboard.tsx [hunk 2]
```diff
@@ -875,7 +875,7 @@ export function AdminReceiverDashboard() {
     }, []);
 
     return (
-        <section className="grid-container">
+        <article>
```
Comment (stephenkao) [inline frontend-react/src/components/Admin/AdminReceiverDashboard.tsx:878]: Articles make more semantic sense than sections for our pages because they're self-contained bodies of content.  Updated a bunch of other pages as well, but we should keep this in mind going forward.

File: frontend-react/src/components/Table/Table.tsx [hunk 3]
```diff
@@ -124,10 +124,7 @@ const Table = ({
         return config.rows;
     }, [config.rows, filterManager?.sortSettings]);
 
-    const wrapperClasses = useMemo(
-        () => `grid-container margin-bottom-10 ${classes}`,
```
Comment (stephenkao) [inline frontend-react/src/components/Table/Table.tsx:128]: If we want to be able to use atomic components anywhere, we can't tie them down to context-specific styles like the `grid-container` here -- that should be taken care of in an ancestor node, and Table/TableFilters should expand to their natural widths inside of that.

**Comment by stephenkao** [comment]
NOTE: This is still subject to review by @jillian-hammer when it's ready on demo3.

**Review by jpandersen87 — APPROVED** [review]
